### PR TITLE
Add tolgee.io

### PR DESCRIPTION
**Pull request by SteffoSpieler**
Should be correct. It's a translation service, that can be self hosted but it's also just available at app.tolgee.io (so... they have it themself hosted?)

### CODE REVIEW

File: img/t/tolgee.io.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+    <path d="M97.16,7.27a16.94,16.94,0,0,0-1.9,24.47,16.36,16.36,0,0,0,5,3.83,3.23,3.23,0,0,1-2.9,5.77,23.14,23.14,0,0,1-11.41-13C73.83,31.1,63.46,37.09,52.82,46.51c-27.44,24.3-34.35,61.74-16.38,85.26-4.57,5.79-8,12.22-8.9,18.69a20.88,20.88,0,0,0,5.62,18c9.18,9.61,21.42,7.13,31.26,5.14,6.58-1.34,12.8-2.6,16.5-.23,3.22,2.07,3.47,3.87,3.61,4.45,2.1,9.32-5.79,13.89-7.67,16.27a1.48,1.48,0,0,0,1.13,2.4c3.48,0,9-1.18,12.34-4.08s7.16-7.9,5.89-16.32c-.08-.5-.18-1-.32-1.58-.86-3.35-3.1-7.57-8.61-11.09-7.72-4.95-17-3.07-25.22-1.41-9.76,2-16,2.85-20.37-1.71a9.13,9.13,0,0,1-2.46-8.19c.54-3.77,2.65-7.89,5.62-11.86,21.71,16.89,56.87,13.47,82.67-9.39a75.34,75.34,0,0,0,20.81-28.09A23.14,23.14,0,0,1,134.8,89a3.23,3.23,0,0,1,6.08-2.19,16.37,16.37,0,0,0,3.2,5.39,16.85,16.85,0,1,0,11.48-28,3.23,3.23,0,0,1-.51-6.44,23.41,23.41,0,0,1,12.88,2.69c2.6-14.08,3.34-31.41-2.06-37.51-4.08-4.61-20.62-8-35.18-7.76A23.48,23.48,0,0,1,130.8,25a3.23,3.23,0,0,1-6.33-1.28A16.94,16.94,0,0,0,97.16,7.27Zm63.25,21a5.29,5.29,0,0,1-.57,6.19c-1.29,1.14-2.72-.51-4.1-2.06s-3.1-3.42-1.81-4.56A5.74,5.74,0,0,1,160.41,28.27Z"/>
+</svg>
```
Comment (Carlgo11) [inline img/t/tolgee.io.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#822b55" viewBox="0 0 147 193"><path d="M70.7 3.7a17 17 0 0 0-2 24.4 16.4 16.4 0 0 0 5 3.9 3.2 3.2 0 0 1-2.8 5.7 23.1 23.1 0 0 1-11.5-13 73.8 73.8 0 0 0-33 18.2C-1.1 67.2-8 104.7 9.9 128.2c-4.5 5.8-8 12.2-8.9 18.7a20.9 20.9 0 0 0 5.7 18c9.1 9.6 21.4 7 31.2 5.1 6.6-1.3 12.8-2.6 16.5-.2 3.2 2 3.5 3.8 3.6 4.4 2.1 9.3-5.8 14-7.6 16.3a1.5 1.5 0 0 0 1 2.4c3.6 0 9-1.2 12.4-4a16.8 16.8 0 0 0 5.6-18c-.9-3.3-3.1-7.6-8.6-11-7.7-5-17-3.2-25.2-1.5-9.8 2-16 2.9-20.4-1.7a9.1 9.1 0 0 1-2.5-8.2c.6-3.8 2.7-7.9 5.7-11.9 21.7 17 56.8 13.5 82.6-9.3a75.3 75.3 0 0 0 20.8-28.1 23.1 23.1 0 0 1-13.5-13.8 3.2 3.2 0 0 1 6-2.2 16.4 16.4 0 0 0 3.3 5.4 16.9 16.9 0 1 0 11.5-28 3.2 3.2 0 0 1-.5-6.4 23.4 23.4 0 0 1 12.8 2.7c2.6-14.1 3.4-31.5-2-37.6-4.1-4.6-20.7-8-35.2-7.7a23.5 23.5 0 0 1 .1 9.8 3.2 3.2 0 0 1-6.3-1.3A17 17 0 0 0 70.7 3.7zm63.2 21a5.3 5.3 0 0 1-.6 6.2c-1.3 1.1-2.7-.5-4-2.1-1.3-1.6-3.2-3.4-1.9-4.6a5.7 5.7 0 0 1 6.5.5z"/></svg>
```

File: entries/t/tolgee.io.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "Tolgee": {
+    "domain": "tolgee.io",
+    "categories": [
+      "developer"
+    ],
+    "tfa": [
+      "totp"
```
Comment (Carlgo11) [inline entries/t/tolgee.io.json:8]: I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by SteffoSpieler** [comment]
Sadly there isn't (yet) documentation about 2fa in tolgee...
But there are some screenshots! First on [Twitter](https://mobile.twitter.com/cyyynthia_/status/1569637468508405761), there's a tweet (with screenshot) by the one who added 2fa.
I for myself can show this screenshot which shows the security screen with a 2fa section:
grafik

**Review by Carlgo11 — APPROVED** [review]

**Comment by SteffoSpieler** [comment]
grafik
Also sadly I've **just** noticed the rank is too low... I've looked at the wrong number ;-;

**Comment by Carlgo11** [comment]
Whoops, looks like the site doesn't meet the requirements of new site listings 🫤

Unfortunately, as the site you want to add is ranked above 200,000 it doesn't meet our guidelines and thus can't be added at the moment. If the site rises in popularity in the future and is ranked below 200,000 on Similarweb, this pull request can be reopened.

To prevent any of your future pull requests from being closed, please make sure your contribution adheres to our [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) and [EXCLUSION.md](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md) before creating a new pull request. Thank you!

**Pull request closed without merging**